spec/index.md

```diff
@@ -3744,10 +3852,10 @@ Conditions
 ```html
 
 S.call_contexts[Ctxt_id].needs_to_respond = false
-∀ CallMessage {origin = FromCanister O, …} ∈ S.messages. O.calling_context ≠ Ctxt_id
-∀ ResponseMessage {origin = FromCanister O, …} ∈ S.messages. O.calling_context ≠ Ctxt_id
-∀ (_ ↦ {needs_to_respond = true, origin = FromCanister O, …}) ∈ S.call_contexts: O.calling_context ≠ Ctxt_id
-∀ (_ ↦ Stopping Origins) ∈ S.canister_status: ∀(FromCanister O, _) ∈ Origins. O.calling_context ≠ Ctxt_id
+∀ CallMessage {origin = FromCanister O, deadline, …} ∈ S.messages. O.calling_context ≠ Ctxt_id ∨ deadline = Expired
```

Comment:
Do we need ` ∨ deadline = Expired` or can we assume that the (expired) call context will be removed before applying this transition?

Reply:
We can't assume that the expired message will be removed. Here we ensure that the appropriate reject response (with `SYS_UNKNOWN`) has been delivered; this is because such responses all use `NoDeadline`, so they can't be ignored in this precondition. But downstream processing of responses can go on.